admin_guide/iptables.adoc

```diff
@@ -11,18 +11,89 @@
 toc::[]
 
 == Overview
-This topic describes how administrators should work with iptables. openshift-sdn takes care of adding the
-necessary iptables rules to make it work. Kubernetes and Docker also manage iptables for port forwarding
-and services.
+There are many system components including {product-title}, Docker, and the Firewall that rely on
```

Comment:
"The Firewall"?  I know what youy mean, but it's not immediately clear.

 How about "the local firewall policies"?